### PR TITLE
Feature/oph24 14 landing page

### PR DESCRIPTION
**Pull request by D33102**
There are still plenty of things to change...

### CODE REVIEW

**Comment by shalluv** [comment]
Please take a look at the [b2c0e00](https://github.com/isd-sgcu/oph66-frontend/pull/9/commits/b2c0e0096a5fc189bce1e65c424dc7d95859ee92) commit.

- Astro recommend to put images in `assets` folder for image enhancing.
- It is better to adjust the spacing by the parent element like padding instead of margin in children element.
- icon use iconify instead

**Comment by shalluv** [comment]
Oh image need to image it first. You can take a look in my commit.

**Comment by D33102** [comment]
> Oh image need to image it first. You can take a look in my commit.

I don't really get this55555

**Comment by shalluv** [comment]
> > Oh image need to image it first. You can take a look in my commit.
> 
> 
> 
> I don't really get this55555

Excuse me for my misspelled, I meant to say "image need to import it first"

File: src/pages/index.astro [hunk 1]
```diff
@@ -1,7 +1,37 @@
 ---
+import { Image } from "astro:assets";
+
 import BaseLayout from "@/layouts/BaseLayout.astro";
+
+import logo from "@/assets/logo.svg";
```
Comment (shalluv) [inline src/pages/index.astro:6]: like this

Comment (D33102) [reply src/pages/index.astro:6]: Thx, I will change it soon.

**Review by shalluv — APPROVED** [review]

**Pull request merged**